Source/JavaScriptCore/wasm/WasmBBQJIT64.cpp

```diff
@@ -5130,7 +5130,9 @@ PartialResult WARN_UNUSED_RETURN BBQJIT::addCallRef(const TypeDefinition& origin
         {
             auto calleeTmp = calleeInstance;
             m_jit.loadPtr(Address(calleePtr, WebAssemblyFunctionBase::offsetOfBoxedWasmCalleeLoadLocation()), calleeTmp);
+            auto branch = m_jit.branchTestPtr(ResultCondition::Zero, calleeTmp);
             m_jit.loadPtr(Address(calleeTmp), calleeTmp);
+            branch.link(&m_jit);
```

Comment:
When does it happen?

Reply:
This happens in cases where we create a WebAssemblyWrapperFunction; since that initializes our load location to 0, we need to branch. I've confirmed that adding these branches is neutral on perf.

Reply:
Does it happen without `useJIT=0`? Or is it happening only when `useJIT=0`? If it is latter, do you know why this happens only when `useJIT=0`?

Reply:
This happens without `useJIT=0` as well; it's just a byproduct of the change to bookkeeping Callee load locations.

Reply:
For `WebAssemblyWrapperFunction`, can we put a thunk calling a function instead of branching? I'm concerning about inserting null-check for each CallRef case.